data-prepper-plugins/common/src/main/java/org/opensearch/dataprepper/plugins/sink/NoopSink.java

```diff
@@ -0,0 +1,30 @@
+package org.opensearch.dataprepper.plugins.sink;
```

Comment:
All new files need a header. Use the following if the code is exclusively new code:

```
/*
 * Copyright OpenSearch Contributors
 * SPDX-License-Identifier: Apache-2.0
 */
```

There is another if you are pulling from another open-source project. Let us know if you need it.